<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Table</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="1">
    <widget class="QGroupBox" name="rulesGrid">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Rules</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QPushButton" name="addBtn">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="deleteBtn">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="folderBtn">
        <property name="text">
         <string>Add folder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTreeView" name="treeView">
        <attribute name="headerDefaultSectionSize">
         <number>0</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QGroupBox" name="tableGrid">
     <property name="title">
      <string>Table</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0">
       <widget class="QLabel" name="nameLbl">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QTextEdit" name="notes"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTableWidget" name="table"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="rulesLbl">
        <property name="text">
         <string>Notes:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="templateLbl">
        <property name="text">
         <string>Template:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="templates"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="name"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="example"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="generateBtn">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
